browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -804,19 +804,11 @@ void RewardsDOMHandler::GetAdsData(const base::ListValue *args) {
   if (ads_service_ && web_ui()->CanCallJavascript()) {
     base::DictionaryValue adsData;
 
-    bool ads_ui_enabled;
     bool ads_enabled = ads_service_->is_enabled();
     int ads_per_hour = ads_service_->ads_per_hour();
 
-    #if BUILDFLAG(BRAVE_ADS_ENABLED)
```

Comment:
is it now disabled somewhere else based on the buildflag?

Reply:
@bridiver not that I am aware of

Reply:
cc @tmancey